rust/kernel/lib.rs

```diff
@@ -28,6 +30,7 @@
 #![deny(clippy::perf)]
 #![deny(clippy::style)]
 #![deny(rust_2018_idioms)]
+#![allow(incomplete_features)]
```

Comment:
Please don't use incomplete features. They can break things in unanticipated ways, and the lint is there to discourage people from using it. We should only use nightly features that are absolutely necessary (or is expected to be stabilised soon).

Reply:
Without these incomplete features, I don't know how to create the table at build time:
`warning: the feature const_evaluatable_checked is incomplete and may not be safe to use and/or cause compiler crashes`
`warning: the feature const_generics is incomplete and may not be safe to use and/or cause compiler crashes`

Reply:
> Without these incomplete features, I don't know how to create the table at build time:

I'll have a try, maybe weekend or next week (couldn't do it this week, 'cuz I just had my jab and are very tired :)